ibis/tests/sql/snapshots/test_compiler/test_table_drop_with_filter/out.sql

```diff
@@ -1,18 +1,18 @@
-SELECT t0.*
-FROM (
-  SELECT t1.`a`
-  FROM (
-    SELECT t3.`a`, t3.`b`, '2018-01-01T00:00:00' AS `the_date`
-    FROM (
-      SELECT t4.*
-      FROM (
-        SELECT t5.`a`, t5.`b`, t5.`c` AS `C`
-        FROM t t5
-      ) t4
-      WHERE t4.`C` = '2018-01-01T00:00:00'
-    ) t3
-  ) t1
-    INNER JOIN s t2
-      ON t1.`b` = t2.`b`
-) t0
-WHERE t0.`a` < 1.0
\ No newline at end of file
+WITH t0 AS (
```

Comment:
This actually looks better.

Reply:
Why does this change as a result of changing the internal schema representation?